Tools/MiniBrowser/mac/WK2BrowserWindowController.m

```diff
@@ -761,9 +766,11 @@ - (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigati
         return;
     }
 
-    if (navigationAction._userInitiatedAction && !navigationAction._userInitiatedAction.isConsumed) {
+    NSURL* url = navigationAction.request.URL;
```

Comment:
Nit: In Objective C code we write this as: `NSURL *url = navigationAction.request.URL`

Reply:
Also: Is a similar change needed in WK1BrowserWindowController for the Legacy WebKit code path?

Reply:
I don't think a similar change is needed for WK1. The API is slightly different and we unconditionally tell the PolicyDecisionListener to go ahead with the navigation in MiniBrowser WK1. I also don't think there's going to be confusion in WK1 mode between Safari and MiniBrowser since Safari is WK2 and there's probably many more divergences in behavior between a WK1 and WK2 browser.